### ISSUE TITLE
Savings/Budget | Feature Request 

### ISSUE THREAD

**Issue by iDoNtCaRe-69** [RetroMusicPlayer/Paisa#219]
First Of All, This Isn't What You Think. This Feature Have A Separate Meaning & Lebel Should Be Changed Later On If Ever Comitted. 

Let's Start With The Fundamental. This Feature Is Dedicated For A Single Purpose, Saving Money For A Specific Reason. For Example, Maybe A New Smartphone, Car, Home, AC, etc.

But The Question Is, Why Is It Different Then Traditional Account System? Well It's A Sub-System Of That Eventually. The Actual Amount Isn't Going Anywhere, It's Staying In The Same Account & If We Deduct It Or Show It As  A Separate Account, Both Way It's Wrong.

**So In Summary:-**
- This Is Not An Account.
- No Deduction From Total Amount.
- Concept Comes From **Gulak**. (Hmm, This Name Sounds Cool!)
- Money Isn't Going Anywhere So Deducting Will Only Confuse Everyone.
- Separate Tab Is Necessary For Additional Feature.

**Comment by h4h13** [comment]
The request seams like goals to me ?

**Comment by iDoNtCaRe-69** [comment]
You Can Say That.

**Comment by DrakeChao** [comment]
> The request seams like goals to me ?

Any update on this? If this festure is being worked on?

**Comment by h4h13** [comment]
Goals are tricky thing to implement maybe for future of Paisa app will be closing this for now
